page-weather/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font-awesome/css/font-awesome.min.css">
+    <script src="https://api-maps.yandex.ru/2.1/?apikey=8334e7a6-6bb1-44c2-a061-4d1f8662219d&lang=en_RU" type="text/javascript">
+    </script>
+    <title>Weather</title>
+</head>
+<body>
+    <div class="wrapper">
+        <img class="background" src="" alt="background"></img>
+        <div class="page-load">
+        <h2 class="intro"></h2>
+            <i class="fa fa-spinner fa-pulse fa-5x" aria-hidden="true"></i>
+        </div>
+        <div class="wrapper__control">
+            <div class="control-block">
+            <button class="switch-image">
+                <i class="fa fa-refresh fa-lg " aria-hidden="true" ></i>
+            </button>
+            <div class="switch-language">
+                <div class="btn-languages">
+                    <button type="button" class="btn-l btn-en">EN</button>
+                    <button type="button" class="btn-l btn-ru">RU</button>
+                </div>
+                <button type="button" class="btn btn-sort"><i class="fa fa-sort-desc " aria-hidden="true"></i>
+                </button>
+            </div>
+            <div class="switch-temperature">
+                <button type="button" class="btn btn-F">°F</button>
+                <button type="button" class="btn btn-C">°C</button>
+            </div>
+        </div>
+            <div class="block-search">
```

Comment:
if you use the _block-search_ as  parent bloc, child block will _block-search__field_